src/arrayMethodJoin.js

```diff
@@ -5,8 +5,29 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let res = '';
+    let separ = separator;
+
+    if (separator === undefined) {
+      separ = ',';
+    }
+
+    for (let i = 0; i < this.length - 1; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        res += separ;
+      }
```

Comment:
Use key word continue here to go to the next iteration and avoid the next if / else statement

Reply:
it doesn't work if I use continue, because it needs to add a separator

Reply:
Add continue after the string with adding separator